app/models/feed.rb

```diff
@@ -47,10 +53,50 @@ def default_runtime_settings?
   end
 
   def published_url
-    if default?
-      "#{podcast.base_published_url}/#{file_name}"
-    else
-      "#{podcast.base_published_url}/#{slug}/#{file_name}"
+    "#{podcast.base_published_url}/#{published_path}"
+  end
+
+  def published_path
+    default? ? file_name : "#{slug}/#{file_name}"
+  end
+
+  def feed_episodes
+    include_in_feed = []
+    feed_max = display_episodes_count.to_i
+    filtered_episodes.each do |ep|
+      next unless include_episode_categories?(ep)
```

Comment:
The categories are stored as a json string attribute, so this filtering is happening in the Ruby code rather than the database query